requirements.txt

```diff
@@ -31,6 +31,6 @@ requests>=2.20
 ruamel.yaml>=0.16,<0.17.18  # See https://github.com/great-expectations/great_expectations/issues/4152
 scipy>=1.6.0
 tqdm>=4.59.0
-typing-extensions>=3.10.0.0,<4.6.0 # Leverage type annotations from recent Python releases
+typing-extensions>=3.10.0.0 # Leverage type annotations from recent Python releases
```

Comment:
Removed this pin due to this fix in pydantic and subsequent release
- https://github.com/pydantic/pydantic/pull/5826
- https://github.com/pydantic/pydantic/releases/tag/v1.10.8